### ISSUE TITLE
Signup Error Messages are inaccurate

### ISSUE THREAD

**Issue by KyleIWS** [sheared-sasquatch/sparkclips#9]
The issues look like this:
![image](https://cloud.githubusercontent.com/assets/19737092/25820023/b1f4c72c-33e4-11e7-86ca-fe1234bc7a6e.png)

Occurs when user signs up with account that already exists. Basically, user is prompted for bad username AND email even though only email is required.

Next issue is elucidated through these two images:
![image](https://cloud.githubusercontent.com/assets/19737092/25819924/5fe4ce32-33e4-11e7-8139-c7c632a9be24.png)
![image](https://cloud.githubusercontent.com/assets/19737092/25819947/6eeaf55a-33e4-11e7-8e94-8f345107bc7e.png)

Basically, the inline-reactive password warning says a password must be 6-100 characters long, but the result of submiting a sign-up form with a password 6 characters long is the error in the second image. So, they need to be consistent. Error message inline should say 8-100 characters
